corehq/apps/user_importer/importer.py

```diff
@@ -389,11 +391,37 @@ def create_or_update_users_and_groups(domain, user_specs, group_memoizer=None, u
                         user.reset_locations(location_ids, commit=False)
 
                 if role:
-                    user.set_role(domain, roles_by_name[role].get_qualified_id())
+                    role = roles_by_name[role].get_qualified_id()
```

Comment:
nit: maybe change to `role_object` (or change the earlier variable to `role_name`) to avoid unexpected type change in the middle of the function?

Reply:
looking closer, maybe this is `role_id`? (I don't actually know what `get_qualified_id` does)